evopy/evopy.py

```diff
@@ -0,0 +1,48 @@
+"""Module used for the execution of the evolutionary algorithm."""
+import numpy as np
+
+from evopy.individual import Individual
+
+
+#pylint: disable=R0913
```

Comment:
Maybe add an explanation here for the why and the what of this suppression?

Reply:
Added one